pkg/conf/git_provider.go

```diff
@@ -1,21 +1,22 @@
 package conf
 
 import (
-	"fmt"
+	"fmt" 
 
 	"github.com/kelseyhightower/envconfig"
 )
 
 type GitProviderConfig struct {
 	Provider           string `envconfig:"GIT_PROVIDER" required:"true"`
 	Token              string `envconfig:"GIT_TOKEN" required:"true"`
-	OrgName            string `envconfig:"GIT_ORG_NAME" required:"true"`
+	Url			       string `envconfig:"GIT_URL" required:"true"`
```

Comment:
```suggestion
	Url			       string `envconfig:"GIT_URL" required:"false"`
```

if not provided and it is gitlab.. should use the SaaS version